apps/explorer/project.json

```diff
@@ -66,6 +66,14 @@
         "passWithNoTests": true
       }
     },
+    "generate-types": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "commands": [
+          "npx openapi-typescript https://raw.githubusercontent.com/vegaprotocol/documentation/main/specs/v0.62.1/blockexplorer.openapi.json --output apps/explorer/src/types/explorer.d.ts --immutable-types"
```

Comment:
question (non-blocking): have you checked out https://github.com/vegaprotocol/frontend-monorepo/issues/2176#issuecomment-1320175206? just curious if you liked this lib better

Reply:
Ah crap - I missed that comment 😢. I'm going to unmark this as closing that ticket and try out orval in comparison in that ticket instead. This works fine, but I don't love some aspects of it (hence, I suppose, the level of commitment of running it with npx).

Reply:
also, on another note, not sure, do any other apps use / need to use the explorer api's return types? i remember having some discussions around fetching data into the wallet from there (currently not the happening). i might consider having the explorer API related things into their own library, like `explorer-client` or something like that. that would give quite some flexibility, easily reusable, even publishable if needed. probably not needed for a first iteration tho.

Reply:
Not currently. It could happen in future. I'm going to close this for now but it is a good point.